app/views/users/index.html.erb

```diff
@@ -1,10 +1,21 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Blog</title>
-  </head>
-  <body>
-    <h1>Users#index</h1>
-    <p>Find me in app/views/users/index.html.erb</p>
-    <p>New Placeholder: Number of posts</p>
-</html>
+<%= stylesheet_link_tag "userindex" %>
+
+<ul class='cardlist'>
+  <% @users.each do |user| %>
+    <li class='card'>
+      <div class='photo'>
+        <% if user.photo.present? %>
+          <%= image_tag(user.photo, alt: user.name, class: "user-photo") %>
+        <% else %>
+          <%= image_tag "user.jpeg", height: 200 %>
```

Comment:
Nice job so far😉😉 I think this piece of code tries to render a default image, in case the user doesn't provide or wants to provide an image, and this is a good implementation👍🏼. However, I am getting an error from the assets pipeline, which indicates that the default image `user.jpeg` has not been defined in the assets pipeline😉. This is because there is no such image as `user.jpeg` in the assets directory😉.

<img width="1056" alt="image" src="https://github.com/DevKaranJ/Blog/assets/64643995/518ce996-8097-4794-ad7f-228ee18cc058">

So what can we do to fix this?🤔 I will suggest two things you can do😉.
1. You can add the default image with the name `user.jpeg` or whatever the name of the image or the extension, you can make the adjustments. this image will be added to the assets path `app/assets/images` and will be rendered when the user doesn't provide an image😉. This is the method I strongly method as it provides a richer user interface even tho it is not required😉.
2. You can take away the condition as regards whether the image is present or not by taking away the else block😉. 

Please, kindly do well to fix the issue, so your app can work just fine👍🏼.